packages/design-system/src/components/Input/Input.tsx

```
@@ -24,14 +25,14 @@ export const Input = ({
     isPasswordInput && isPasswordVisible ? "text" : type;
 
-  const containerClassName = isAddressInput
+  const baseContainerClassName = isAddressInput
     ? "rounded-[999px] bg-gray-200"
     : cn(
-      "rounded-[8px] border",
-      error
-        ? "border-accent"
-        : "border-gray-500 focus-within:border-primary"
-    );
+        "rounded-[8px] border",
```

ESLint

[error] Expected indentation of 6 spaces but found 8.

(indent)